modules/about-microshift.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// microshift/understanding-microshift.adoc 
+
+[id="con-about-microshift_{context}"]
+= About MicroShift
+
+Working with low-resource field environments and hardware presents many challenges not experienced with cloud computing. {product-title} enables you to solve problems for edge devices deployed in the field by:
+
+* Overcoming the operational challenge of minimal system resources, for example, a {op-system-chip} with {op-system-ram}.
```

Comment:
I'm not sure of the usual practice with attributes like this. When we add ARM support, would we want to add another set of attributes for providing an example of that hardware?

Reply:
We could; I wanted to try it out and see what we think. My thinking was we might want attributes if we want to showcase changing or processor-specific minimums easily; and because I am not sure if we'll list the requirements elsewhere so they would be more accurately updated if made into attributes.

Reply:
OK, that makes sense. Let's see how it works out.